pkgdb/src/resolver/lockfile.cc

```diff
@@ -460,27 +460,20 @@ lockedPackageFromCatalogDescriptor( const nlohmann::json & jfrom,
   pkg.info     = jfrom;
   pkg.input    = LockedInputRaw();
 
-  pkg.input.url   = jfrom["locked_url"];
-  pkg.input.attrs = nix::fetchers::Attrs();
+  pkg.input.url = jfrom["locked_url"];
+
   // These attributes are needed by the current builder, and not included in the
   // descriptor. This will not always be true, but also may not be required to
   // build depending on the path taken for future environment builds.
-  if ( std::string supportedUrl = "github:NixOS/nixpkgs";
+  if ( std::string supportedUrl = "github:flox/nixpkgs";
        pkg.input.url.substr( 0, supportedUrl.size() ) != supportedUrl )
     {
       throw InvalidLockfileException(
         "unsupported lockfile URL for v1 lockfile",
         "must begin with " + supportedUrl );
     }
```

Comment:
this seems to imply that we now _only_ support `flox/nixpkgs`?! Since its a fork there is currently no technical reason to restrict this to flox/nixpkgs.

Reply:
Yeah I think the idea is that this is a hack we may revisit. And since it's a hack, we might as well scope it down as much as we can. When we revisit, we can discuss proper support of other flakes

Reply:
I see but wouldn't that be breaking everyone's floxes unless they activate the catalog feature?

Reply:
Ah, this code path is only for the "new" lock file though right?